.github/workflows/rbc_test.yml

```diff
@@ -150,19 +150,6 @@ jobs:
             mamba create -n heavydb-env omniscidb=${{ matrix.heavydb-version }}*=*_cpu -c conda-forge
           fi
 
-      - name: Build heavydb docker image and run it [docker]
-        shell: bash -l {0}
-        env:
-          HEAVYDB_DOCKER_IMAGE: ${{ matrix.docker-image }}
-        if: matrix.os == 'ubuntu-latest' && matrix.heavydb-from == 'docker'
-        run: |
-          # force docker-compose installation
-          mamba create -n docker docker-compose -c conda-forge
-          # run docker services
-          mamba run -n docker docker-compose build
-          mamba run -n docker docker-compose up -d
-          sleep 20
-
```

Comment:
Oh, sorry. Maybe I wasn't clear in my last comment. We should keep the instructions to run the heavydb docker image.